js/src/contract/item.ts

```diff
@@ -410,12 +410,14 @@ export class ItemProgram {
       this.program.provider as AnchorProvider
     ).connection.getAccountInfo(itemClass);
 
+    // QUESTION: data is optional, what should we do if it's not populated?
```

Comment:
In cases like these, I would either throw or return `null`. Given this is a fetch function in a low level library, throwing would not be appropriate. I would return `null` to signify not found.

To give you an alternate example of when I would throw. If you were using this fetch call from a **client**, and the client needed this item class to exist to continue, then I would throw from the client after making the fetch call if fetching returned `null`.